uber/templates/preregistration/preregbase.html

```diff
@@ -23,7 +23,9 @@
                 onClick: function () {
                     $('.group_fields').addClass('hide');
                     if ($.field('badge_type').val() == '{{ c.PSEUDO_GROUP_BADGE }}') {
-                        $('input[name="badge_type"]').val('{{ c.ATTENDEE_BADGE }}');
+                        if ($('input[name="badge_type"]').val() != '{{ c.STAFF_BADGE }}') {
```

Comment:
This and the other changes are good, but it needs to be expanded a bit. If we just merge this, we'll find the same bug with Guest badges later down the road.

Luckily we have a nice property called `PREREG_BADGE_TYPES` that we can use. We should check against it instead, like this:
`if ($('input[name="badge_type"]').val() in {{ c.PREREG_BADGE_TYPES }}) {
  $('input[name="badge_type"]').val('{{ c.ATTENDEE_BADGE }}');
}`

That was it only changes the badge if the attendee's badge type is either an attendee badge, one of the pseudo badges, or a badge_type_price badge (a special kind of badge that MAGFest doesn't use).